inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/recommender/NamedEntityLinker.java

```diff
@@ -131,7 +131,7 @@ public void setProject(Project project)
     {
         List<List<List<AnnotationObject>>> result = new ArrayList<>();
 
-        for (List<T> sentence : inputData) {
+        inputData.parallelStream().forEach(sentence -> {
```

Comment:
Did you measure whether it is actually faster?

Reply:
Yes, it was twice as fast. Evaluated with 16 sentences annotated with NEs.
[NER-de-sample.tsv.conll.zip](https://github.com/inception-project/inception/files/2000127/NER-de-sample.tsv.conll.zip)